### ISSUE TITLE
Update shows wrong version when running a beta

### ISSUE THREAD

**Issue by filipw** [filipw/dotnet-script#414]
This is what I get when running the `--info` command on `0.29.0-beta1`

```
Version             : 0.29.0-beta1
Install location    : Z:\Documents\dev\dotnet-script\src\Dotnet.Script\bin\Debug\netcoreapp3.0
Target framework    : netcoreapp3.0
.NET Core version   : 3.0.0-preview-27122-01
Platform identifier : win
Runtime identifier  : win10-x64
Version 0.28.0 is now available
Depending on how dotnet-script was installed, execute on of the following commands to update.
Global tool : dotnet tool update dotnet-script -g
Chocolatey  : choco upgrade Dotnet.Script
Powershell  : (new-object Net.WebClient).DownloadString("https://raw.githubusercontent.com/filipw/dotnet-script/master/install/install.ps") | iex
```

it doesn't seem to understand that `0.29.0-beta1` > `0.28.0`

**Comment by seesharper** [comment]
This is because we , well okay 😀, I took the super simple approach here where we look at the latest version available on GitHub releases and compare it to the current version. By compare I mean that we only look to see if they are different. We do no attempt to reason about the latest version being "greater" than the current. This works like a charm until you actually download the repo and bumps the version number and then issue a `dotnet --info` with the binaries built locally. So for the end user this will never be a problem. Another upside of this simple approach is that we don't need any "NuGet" dependencies to compare versions. The intent was to keep `Dotnet.Script.Core` as clean as possible and hence I decided not to take that dependency.
